package.json

```diff
@@ -4,22 +4,28 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "NODE_ENV=dev nodemon ./src/index.js --exec babel-node --",
+    "start": "./src/index.js --exec babel-node --",
     "build": "babel src -d dist ",
     "eslint": "eslint --ignore-path .gitignore .",
-    "dev": "nodemon --exec babel-node ./src/index.js",
-    "test": "NODE_ENV=testing nyc mocha ./src/tests/*.js  --timeout 200000 --exit",
-    "coverage": "nyc npm test  && npm run coveralls-coverage"
+    "test": "NODE_ENV=testing npm run dropTables && NODE_ENV=testing npm run db-migrate && NODE_ENV=testing nyc --reporter=text --reporter=html --require @babel/polyfill --require @babel/register mocha ./src/tests/*.js  --timeout 200000 --exit",
+    "coverage": "nyc npm test  && npm run coveralls-coverage",
+    "dev-start": "nodemon --exec babel-node ./src/index.js",
+    "create-db": "npx sequelize-cli model:generate --name users --attributes firstName:string,lastName:string,email:string,password:string && sequelize db:migrate",
```

Comment:
Remember to change this script because we have added two more fields in database(isVerified, signupType), check it out on my PR of signup